### DISCUSSION TITLE
Welcome to Genie.jl Discussions!

### DISCUSSION THREAD

**Discussion by essenciary** [GenieFramework/Genie.jl#459] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

While you're here, please star our repo 🌟 - it helps us fund the project and provide better support and ever cooler features! Thank you! 

For live chats with users and maintainers, come say "Hi" on our [Discord](https://discord.gg/TSuy5md6). Looking forward to meeting you.

**Comment by AbhimanyuAryan** [comment]
wow thanks @essenciary
